infrastructure/deploy.sh

```diff
@@ -113,7 +113,7 @@ if [[ ! -f terraform.tfstate ]]; then
     # Until terraform plan supports -var-file the plan is wrong.
     # terraform plan
 
-    terraform apply -var-file=environments/$env.tfvars -auto-approve > /dev/null
+    terraform apply -var-file=environments/$env.tfvars -auto-approve
```

Comment:
I assume this is changed because we don't need the "security" of devnull anymore?

Reply:
Maybe we can check something in the environment to determine this instead? I don't want to accidentally bump into this again.